pkgs/applications/science/robotics/betaflight-configurator/default.nix

```diff
@@ -0,0 +1,53 @@
+{stdenv, fetchurl, unzip, runtimeShell, makeDesktopItem, nwjs }:
+
+let
+  strippedName = "betaflight-configurator";
+  desktopItem = makeDesktopItem {
+    name = strippedName;
+    exec = strippedName;
+    icon = "${strippedName}-icon.png";
+    comment = "Betaflight configuration tool";
+    desktopName = "Betaflight Configurator";
+    genericName = "Flight controller configuration tool";
+  };
+in
+stdenv.mkDerivation rec {
+  name = "${strippedName}-${version}";
+  version = "10.5.1";
+  src = fetchurl {
+    url = "https://github.com/betaflight/betaflight-configurator/releases/download/${version}/${strippedName}_${version}_linux64.zip";
+    sha256 = "1wwyl0f2lxsznldjlqdk2b2zzrvvz7lj6c06zq6xzcb1rh971y30";
```

Comment:
I get mismatched hashes :
```
  wanted: sha256:1wwyl0f2lxsznldjlqdk2b2zzrvvz7lj6c06zq6xzcb1rh971y30
  got:    sha256:1l4blqgaqfrnydk05q6pwdqdhcly2f8nwzrv0749cqmfiinh8ygc
```
solved by the suggested hash : 
```
nix-prefetch-url https://github.com/betaflight/betaflight-configurator/releases/download/10.5.1/betaflight-configurator_10.5.1_linux64.zip 
[86.5 MiB DL]
path is '/nix/store/b21jrpsxxnwm89j6kr736016dnl4bdvl-betaflight-configurator_10.5.1_linux64.zip'
1l4blqgaqfrnydk05q6pwdqdhcly2f8nwzrv0749cqmfiinh8ygc
```

Reply:
Oh no! I confused this :( Fix incoming!